docs/en/enterprise-edition/content-collections/runtime-security/agentless-scanning/configure-accounts/frag-start-agentless-scan.adoc

```diff
@@ -1,27 +1,65 @@
-[#start-agentless-scan]
+[#agentless-scanning-for-cloud-accounts]
 [.task]
-== Start an Agentless Scan
+=== Agentless Scanning for Cloud Accounts
 
-Agentless scans start immediately after onboarding the cloud account.
-By default, agentless scans are performed every 24 hours, but you can change the interval on the *Manage > System > Scan* page under *Scheduling > Agentless*.
+When new cloud accounts are onboarded to Prisma Cloud with the "Agentless Scanning" option enabled, scanning starts immediately. If this option is disabled before onboarding, scanning does not occur for those accounts.
+
+When you enable agentless scanning for existing accounts in Prisma Cloud, it does not trigger an immediate scan. Instead, the account is included in the next scan cycle, which occurs every 24 hours by default.
+
+To modify the scan cycle, see <<modifying-the-agentless-scan-interval>>
+
+[#editing-agentless-scan-settings]
+=== Editing Agentless Scan Settings
+
+For existing cloud accounts that have agentless scan settings disabled, you can now modify the settings directly in Runtime Security > Manage > Cloud accounts page. 
```

Comment:
add a sentence that by editing accounts that are disabled, you can then safely enable them and the scan will trigger with the correct settings

Reply:
I have changed the para to "You can safely enable agentless scanning settings for disabled accounts on the *Runtime Security* > *Manage* > *Cloud accounts* page. After enabling the settings on this page, agentless scanning will trigger with the correct configuration.
"